src/scripts/cron-update.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xe
+
+# Clone the repo into a temporary directory
+# TMPDIR=$(mktemp -d -t opencovid-$(date +%Y-%m-%d-%H-%M-%S)-XXXX)
+TMPDIR="/tmp/opencovid-$(date +%Y-%m-%d-%H-%M-%S)-XXXX"
+git clone https://github.com/open-covid-19/data.git --single-branch -b main "$TMPDIR/opencovid"
+
+# Download the intermediate files into the working directory
+mkdir -p "$TMPDIR/opencovid/output/snapshot"
+mkdir -p "$TMPDIR/opencovid/output/intermediate"
+gsutil -m cp -r gs://covid19-open-data/snapshot "$TMPDIR/opencovid/output/"
+gsutil -m cp -r gs://covid19-open-data/intermediate "$TMPDIR/opencovid/output/"
+
+# Install dependencies and run the update command in a Docker instance
+docker run -v "$TMPDIR/opencovid":/opencovid -w /opencovid -i python:3.8-buster /bin/bash -s <<EOF
```

Comment:
FYI, most of this section I don't understand so can't really review unless I read up on docker and cloud stuff.

Reply:
It runs all the commands between this line and EOF inside of a Docker container, which ensures a stable environment (same version of Python, libraries installed from pip, etc.)